docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,29 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding formulae
+Only one maintainer is necessary to approve and merge the addition of a new formula which passes CI. However, if the formula addition
+is controversial the maintainer who adds it will be expected to fix issues that arise with it in future.
+
+### Removing formulae
+Formulae that:
+
+- work on at least 2/3 of our supported macOS versions in the default Homebrew prefix
+- do not require patches rejected by upstream to work
+- do not have known security vulnerabilities/CVEs for the version we package
+- are shown to be still installed by users in our analytics with a `BuildError` rate of <25%
+
+
+should not be removed from Homebrew. The exception to this rule are [versioned formulae](Versions.md) for which there are higher standards of usage and a maximum number of versions for a given formula.
+
```

Comment:
Cask may also benefit from these sections. Maybe add a "Updating formulae" section so that we know how/when to merge updates safely?

Reply:
@amyspark Will do 👍